packages/neutrino/bin/neutrino

```diff
@@ -64,10 +70,12 @@ const normalizeOptions = (args) => {
   const pkg = getPackageJson();
   const options = pathOr({}, ['neutrino', 'options'], pkg);
   const env = args.inspect ? getNodeEnv(args._[0], args.env) : args.env;
+  const debug = args.debug ? args.debug : options.debug;
```

Comment:
This variable is only used to define an object property, so we can inline it into the object itself.

```js
debug: args.debug || options.debug
```